### PR TITLE
chore(eventhandler): include options object in event handler

### PR DESCRIPTION
**Pull request by esatterwhite**
Includes a common options object to be passed into the event handler constructor and propogated through its factory function.

This makes adding additional customer properties and configration easier to manage between the various repos and minimizes the need to explicitly expose option in every project

### CODE REVIEW

**Review by alewitt2 — DISMISSED** [review]
lgtm

**Comment by esatterwhite** [comment]
Is there a way to re-trigger this build?

**Comment by alewitt2** [comment]
i think you just have to push an empty commit
```
git commit --allow-empty -m "Trigger Build"
```

**Comment by esatterwhite** [comment]
@alewitt2 done

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "lts/*"
+  - "lts/gallium"
```
Comment (alewitt2) [inline .travis.yml:4]:
```suggestion
  - "16"
```

**Comment by esatterwhite** [comment]
@alewitt2 Seems like the `DOCKERHUB_TOKEN` environment variable isn't set correctly or is missing from travis

**Comment by alewitt2** [comment]
hmm thats very odd

**Comment by alewitt2** [comment]
https://github.com/razee-io/Kubernetes-util/pull/291

**Pull request closed without merging**